### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by solog0039**
In this project:
>Github work flow created
>HTML file created
>CSS file created
>Linter installed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+[tiwtter](https://twitter.com/Solomon57320119)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumps if you like this project.
+## Acknowledgments
+Microverse
+## License
+This project is MIT License licensed.
```
Comment (Ibizugbe) [inline README.md]: I can see you made a great effort creating your readme👍, to make our projects look more professional, It is a requirement that we make use of the [Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)😉 kindly make use the given [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) to create your readme.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @solog0039 👋, 
PointThumbsUpGIF
Good job so far!🤗🤗
There are some issues that you still need to work on to go to the next project but you are almost there!👍

## Highlights
- [ ] All linter checks are passing.
- [ ] The right github flow method has been used.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL]  I think you can make your commit messages more descriptive in your future project as this makes your project appear more professioal. I will suggest you take a look at [this article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) so as to help you improve.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+  [tiwtter](https://twitter.com/Solomon57320119)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumbs up if you like this project.
+## Acknowledgments
+Microverse
+## License
+This project is Apache License licensed.
```
Comment (wayungi) [inline README.md]: - [ ] Please as requested by the previous reviewer, use [this README template](https://github.com/microverseinc/readme-template). Using a professional README template makes your repo professional.

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @solog0039 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights ⭐ ⭐ 
- [ ] Correct GitHub flow
- [ ]  All linter checks passing


## Required Changes ♻️
 - [ ] Please use this [README](https://github.com/microverseinc/readme-template)  for all your README files and do not remove any sections except for the FAQs 📖 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL ] You can also write better commit messages using the imperative tense. It is better to commit after every feature. For example, when you add a navigation bar, you can commit with a message like **Add nav bar**. Please remember to do this in your next commit messages 👏🏾 
 
- [OPTIONAL] Please create a file called **test.md** and add the following content to that file "_This file should be ignored by git_" 👏🏾 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @solog0039 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## Highlights 
:+1: no linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**